apps/editor.planx.uk/src/pages/FlowEditor/lib/store/settings.ts

```diff
@@ -45,6 +46,8 @@ export const settingsStore: StateCreator<
 
   flowStatus: undefined,
 
+  flowSummary: undefined,
```

Comment:
non-blocking - Here's the alternative pattern I was outlining on the call yesterday - 

- Keep the additional property on on the fetch, and updated type as you have done
- Don't add flowSummary to the store - it's already in the Apollo cache
- Add a query where you need it to fetch flowId and flowSummary using Apollo's `useQuery()`
  - A network request won't fire
  - Apollo will match on flowId and return the flow summary

Reply:
This is for sure non-blocking as there's some more work to be done here around the graphql client first (type policies) as well as making some common utilities (`useTeam()`, `useFlow()` etc) that I want to set up.